src/styles/blocks/header.scss

```diff
@@ -0,0 +1,153 @@
+.header {
+  &__container {
+    @include onLaptop {
+      width: calc(100% - 100px);
+      margin: 0 50px;
+    }
+
+    @include onDesktop {
+      width: 100%;
+      margin: 0;
+    }
+  }
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 15px 0 12px;
+
+    @include onTablet {
+      margin: 27px 0 36px;
+    }
+
+    @include onLaptop {
+      margin: 40px 0 80px;
+    }
+  }
+
+  &__button-open {
+    transition: all $transition-duration;
```

Comment:
don't add a transition for `all` styles. Add transition just for a style that will change smoothly. Fix all transitions, I see so many issues like this 😢